src/main/resources/assets/origins/lang/es_es.json

```
@@ -1,170 +1,243 @@
+  
+  "power.origins.pumpkin_hate.name": "Cucurbitofobia",
+  "power.origins.pumpkin_hate.description": "Le tienes miedo a las calabazas, por buenos motivos.",
+  
+  "power.origins.extra_reach.name": "Cuerpo Esbelto",
+  "power.origins.extra_reach.description": "Puede alcanzar bloques y mobs de más lejos.",
+  
+  "power.origins.shulker_inventory.name": "Acumulador",
+  "power.origins.shulker_inventory.description": "Tienes 9 espacios de inventario extra, los cuales se mantienen después de morir.",
+  
+  "power.origins.natural_armor.name": "Armadura Natural",
+  "power.origins.natural_armor.description": "Incluso sin armadura, tu piel te proporciona protección.",
+  
+  "power.origins.more_exhaustion.name": "Gran Apetito",
+  "power.origins.more_exhaustion.description": "Te da hambre más rápido.",
+  
+  "power.origins.no_shield.name": "Manos Deformes",
+  "power.origins.no_shield.description": "La forma de tus manos no te permite cubrirte con escudos.",
+  
+  "power.origins.strong_arms.name": "Brazos Fuertes",
+  "power.origins.strong_arms.description": "Eres lo suficientemente fuerte como para romper piedra con tus manos.",
+  
+  "power.origins.translucent.name": "Transparente",
+  "power.origins.translucent.description": "Tu piel es transparente.",
+  
+  "power.origins.end_spawn.name": "Habitante del End",
+  "power.origins.end_spawn.description": "Tu aventura empieza en el End.",
+  
+  "power.origins.master_of_webs.name": "Maestro de redes",
+  "power.origins.master_of_webs.description": "Puedes pasar a través de telarañas sin problemas y puedes hasta escalarlas. Cuando golpeas a un enemigo cuerpo a cuerpo, se quedará atrapado en una telaraña por un rato. Mobs no-artrópodos que se queden atrapados en telarañas serán más fáciles de ver para ti. Puedes fabricar telaraña con hilo.",
+  
+  "power.origins.lay_eggs.name": "Ovíparo",
+  "power.origins.lay_eggs.description": "Todas las mañanas cuando despiertes, pondrás un huevo.",
+  
+  "power.origins.phantomize.name": "Forma Fantasma",
+  "power.origins.phantomize.description": "Puedes cambiar entre forma fantasma y humana a voluntad, pero si te agotas lo suficiente para no poder correr, vuelves a tu forma humana.",
+  
+  "power.origins.like_air.name": "Como el Viento",
+  "power.origins.like_air.description": "Tu velocidad en la tierra se mantiene igual en el aire.",
+  
+  "power.origins.velvet_paws.name": "Patas Acolchadas",
+  "power.origins.velvet_paws.description": "Las almohadillas en tus patas hacen que tus pisadas no causen vibraciones, lo que hace que no te puedan detectar los Sensores de Sculk.",
+  
+  "category.origins": "Orígenes",
+  "key.origins.primary_active": "Poder Activo (Principal)",
+  "key.origins.secondary_active": "Poder Activo (Secundario)",
+  "key.origins.view_origin": "Ver Origen",
+  
+  "text.autoconfig.origins.title":   "origins Configuration",
```

Please fix the case here.
```suggestion
  "text.autoconfig.origins.title":   "Origins Configuration",
```